Integrations/integration-Threat_Grid.yml

```diff
@@ -308,6 +309,9 @@ script:
             args,
             filename
         );
+        if (filename){
+            return rawResponse;
```

Comment:
I think its not necessary to call `sendRequest` if `!filename`
What happen if `!filename`?
Can you elaborate on this bug?